src/odemis/acq/fastem.py

```diff
@@ -266,6 +279,62 @@ def __init__(self, scanner, multibeam, descanner, detector, stage, ccd, beamshif
         # Threading event, which keeps track of when image data has been received from the detector.
         self._data_received = threading.Event()
 
+    def pre_calibrate(self):
```

Comment:
I would prefer to have the run function first after the init and then all the functions that are called in the run function. What do you think?